docs/source/en/tasks/document_question_answering.mdx

```diff
@@ -0,0 +1,486 @@
+<!--Copyright 2023 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+specific language governing permissions and limitations under the License.
+-->
+
+# Document Question Answering
+
+[[open-in-colab]]
+
+Document Question Answering, also referred to as Document Visual Question Answering, is a task that involves providing
+answers to questions posed about document images. The input to these models is typically a combination of an image and
+a question, and the output is an answer expressed in natural language. These models utilize multiple features, including
+text, the positions of words (bounding boxes), and the image itself.
```

Comment:
It's a bit weird that you start with "these models" => which models?

Reply:
It is described the same way as in the Task description.  https://huggingface.co/tasks/document-question-answering